<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="databaseTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>1161</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>0</y>
        <width>1171</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="OpenDBButton">
         <property name="text">
          <string>Open Database</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>folder.svg</normaloff>folder.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PlatformDropdown">
         <item>
          <property name="text">
           <string>Instagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Facebook</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Twitter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Snapchat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TikTok</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DBSearchBox">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SearchDBButton">
         <property name="text">
          <string>Search Database</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>search.svg</normaloff>search.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="DBSearchProgbar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Other</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionOpen_Database"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGithub"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
